src/plugins/orangehrmRecruitmentPlugin/Api/CandidateHistoryAPI.php

```diff
@@ -151,22 +153,24 @@ public function getOne(): EndpointResult
         $this->throwRecordNotFoundExceptionIfNotExist($candidateHistoryRecord, CandidateHistory::class);
         $vacancy = $candidateHistoryRecord->getVacancy();
         $disabled = false;
-        if ($candidateHistoryRecord->getAction() === WorkflowStateMachine::RECRUITMENT_APPLICATION_ACTION_SHORTLIST &&
-            (
-                !is_null($vacancy) &&
-                $vacancy->getHiringManager()->getEmpNumber() !== $this->getAuthUser()->getEmpNumber()
-            )
-        ) {
-            $rolesToExclude = ['HiringManager'];
-            $allowedWorkflowItems = $this->getUserRoleManager()->getAllowedActions(
-                WorkflowStateMachine::FLOW_RECRUITMENT,
-                CandidateService::STATUS_MAP[WorkflowStateMachine::RECRUITMENT_APPLICATION_ACTION_ATTACH_VACANCY],
-                $rolesToExclude
-            );
-            $disabled = !in_array(
-                WorkflowStateMachine::RECRUITMENT_APPLICATION_ACTION_SHORTLIST,
-                array_keys($allowedWorkflowItems)
-            );
+        if ($vacancy->getHiringManager()->getEmpNumber() !== null) {
```

Comment:
seems these API changes are not covered in API testing. Seems API changes are backward compatible, as nothing failed